src/cljs/athens/db.cljs

```diff
@@ -513,7 +557,7 @@
   (fnext (drop-while #(not (pred %)) xs)))
 
 
-(d/listen! dsdb :history
+#_(d/listen! dsdb :history
```

Comment:
causing app to crash, probably due to difference in spec between posh/ds. commented out as short fix just to get app running to test